convert2rhel/unit_tests/systeminfo_test.py

```diff
@@ -68,9 +78,33 @@ def tearDown(self):
         if os.path.exists(unit_tests.TMP_DIR):
             shutil.rmtree(unit_tests.TMP_DIR)
 
+    @unit_tests.mock(tool_opts, "no_rpm_va", True)
+    def test_generate_modified_rpms_with_no_rpm_va(self):
+        self.assertEqual(system_info.generate_modified_rpms(), None)
```

Comment:
This is too imprecise, from this test we can't tell whether it respects `no_rpm_va` or simply returns no differences. Change this to be similar to:

https://github.com/oamg/convert2rhel/pull/90/files#diff-b42c590ebd33c41f1d6d03b3372d8da0R120-R127

Reply:
Still applies after refactor to `log_modified_rpms_diff()`. Too imprecise